docs/resources/meeting_conferences.md

```diff
@@ -0,0 +1,526 @@
+    time, the system will select Video by default).
+  + **Data**: Multimedia (If omitted, the system configuration will determines whether to automatically add **Data**).
+
+* `is_auto_record` - (Optional, Int) Specifies whether the conference automatically starts recording, it only takes
+  effect when the recording type is:
+  + **1**: Automatically start recording.
+  + **0**: Do not start recording automatically.
+
+  The default is **0** (not to start automatically).
+
+* `encrypt_mode` - (Optional, Int) Specifies the conference media encryption mode.
+  **0**: Adaptive encryption.
+  **1**: Force encryption.
+  **2**: Do not encrypt.
+
+  The default value is populated by enterprise-level configuration.
+
+* `language` - (Optional, String) Specifies the default language of the conference, the default value is defined by the
+  conference cloud service. For languages supported by the system, it is passed according to the RFC3066 specification.
+  The valid values are as follows:
+  + **zh-CN**: Simplified Chinese.
+  + **en-US**: US English.
+
+* `timezone_id` - (Optional, Int) Specifies the time zone information of the conference time in the conference
+  notification. For time zone information, refer to the [time zone mapping relationship](#time_zone_mapping).
+
+  -> For example: the timeZoneID `26`, time in the conference notification sent through HUAWEI CLOUD conference will be
+  marked as "2021/11/11 Thursday 00:00 - 02:00 (GMT) Greenwich Standard When: Dublin, Edinburgh, Lisbon, London".
+  For an aperiodic conference, if the conference notification is sent through a third-party system, this field does not
+  need to be filled in.
+
+* `record_type` - (Optional, Int) Specifies the recording type.
+  + **0**: Disabled.
+  + **1**: Live broadcast.
+  + **2**: Record and broadcast.
+  + **3**: Live + Recording.
+
+  Default is **0** (disabled).
+
+* `live_address` - (Optional, String) Specifies the mainstream live broadcast address, with a maximum of 255 characters.
+  Only availabe if `record_type` is **2** or **3**.
+
+* `aux_address` - (Optional, String) Specifies the auxiliary streaming address, the maximum length is 255 characters.
+  Only availabe if `record_type` is **2** or **3**.
+
+* `is_record_aux_stream` - (Optional, Int) Specifies whether to record auxiliary stream.
+  + **0**: Do not record.
+  + **1**: Record.
+
+  Only availabe if `record_type` is **2** or **3**, and the default value is **0**.
+
+* `record_auth_type` - (Optional, Int) Specifies the recording authentication method.
+  **0**: Viewable/downloadable via link.
+  **1**: Enterprise users can watch/download.
+  **2**: Attendees can watch/download.
+
+  Only availabe if `record_type` is **2** or **3**.
+
+* `participant_number` - (Optional, Int) Specifies the number of parties in the conference, the maximum number of
+  participants in the conference.
+  **0**: Unlimited.
+  **Greater than 0**: the maximum number of participants in the conference.
```

Comment:
this is a little confusing, how about just use `defaults to: 0 (Unlimited)`

Reply:
good idea